src/components/Accordion/_Accordion.scss

```diff
@@ -33,7 +33,7 @@
         }
     }
 
-    input {
+    > input {
```

Comment:
Can we instead use a BEM block here instead of styling an `input` directly?

Reply:
yes, let's do that instead!